code/modules/clothing/under/jobs/civilian/civilian.dm

```diff
@@ -208,3 +214,17 @@
 	body_parts_covered = CHEST|GROIN|ARMS
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
+
+/obj/item/clothing/under/rank/civilian/lawyer/galaxy
```

Comment:
I would prefer if we avoided "base" types that aren't actually useful - rather one of blue or red is the base outright

Reply:
fixed